content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "Using wolfictl to Manage Security Advisories"
+linktitle: "Managing Advisories"
+aliases: 
+type: "article"
+description: "Guide on how to use the wolfictl tool to create, update, and manage security advisories"
+date: 2024-08-05T20:23:51+00:00
+lastmod: 2024-08-05T20:23:51+00:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "security-advisories"
+weight: 030
+toc: true
+---
+
+Chainguard operates its own [Security Advisories](https://images.chainguard.dev/security/) page to alert users about the status of vulnerabilities found in Chainguard Images. To maintain this database, we use [`wolfictl`](https://github.com/wolfi-dev/wolfictl/), a tool which is used to work with the [Wolfi un-distro](https://github.com/wolfi-dev/).
+
+In this guide, we will walk through using `wolfictl` to create an advisory for a vulnerable package. We’ll also learn how to update this advisory as more information about the vulnerability is disclosed over time. To follow along, you will need to have [‘git’](https://git-scm.com/) and the [go programming language](https://go.dev/dl/) installed on your machine. 
+
+This guide will focus on the packages and advisories issued for Wolfi.
+
+
+## How to Install `wolfictl`
+
+To work with security advisories, we will need to install the `wolfictl` tool to our machine. First, execute the following command in your terminal to clone the `wolfictl` repository locally, and navigate to it. 
```

Comment:
```suggestion
To work with security advisories, you will need to install the `wolfictl` tool onto your machine. First, execute the following command in your terminal to clone the `wolfictl` repository locally, and navigate to it. 
```
This is why I lean on "you" instead of "we" - it isn't *our* computer, it's their computer